.coafile

```diff
@@ -1,4 +1,3 @@
 [python]
 bears = PEP8Bear,PyFlakesBear,PySafetyBear,PyromaBear,RadonBear
 files = **.py
-
```

Comment:
https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-file